tests/messages/test_pofile.py

```diff
@@ -1014,3 +1014,10 @@ def test_issue_1087():
 "Language: \n"
 ''')
     assert pofile.read_po(buf).locale is None
+
+def test_issue_1134():
+    buf = StringIO('''msgid "this is an invalid po file"
+msgstr "hello world"
+msgid "msgid without str"''')
+    with pytest.raises(pofile.PoFileError):
+        pofile.read_po(buf, abort_invalid=True)
```

Comment:
I'd also add a test when `abort_invalid` is False and maybe one test when the input includes `msgid_plural` e.g.

```
msgid "foo"
msgid_plural "foos"
```

(Also, there's a missing newline at the end of this file ;))